### PR TITLE
fix(@umi/utils): add the windows support

### PR DESCRIPTION
**Pull request by chenliangngng**
windows和mac行为不一致，原有方案在windows下会报错。
close https://github.com/umijs/umi-next/issues/391

### CODE REVIEW

File: packages/utils/src/npmClient.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export const installWithNpmClient = ({
   cwd?: string;
 }): void => {
   const { spawnSync } = require('child_process');
```
Comment (fz6m) [inline packages/utils/src/npmClient.ts:33]:
```suggestion
  const { sync } = require('cross-spawn');
```

**Comment by fz6m** [comment]
You can use `cross-spawn` cross platform execute command

File: packages/utils/src/npmClient.ts [hunk 2]
```diff
@@ -31,7 +31,7 @@ export const installWithNpmClient = ({
...
-  const npm = spawnSync(npmClient, [npmClient === 'yarn' ? '' : 'install'], {
+  const npm = spawnSync(process.platform ==='win32'? (npmClient + '.cmd') : npmClient, [npmClient === 'yarn' ? '' : 'install'], {
```
[the remark this answers is not in the document]
Comment (chenliangngng) [reply packages/utils/src/npmClient.ts]: done

**Comment by bzp2010** [comment]
looks good!

**Pull request merged**